Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -539,37 +539,16 @@ namespace OpenInfraPlatform {
 								if(hollow->InnerFilletRadius) {
 									r2 = hollow->InnerFilletRadius * length_factor;
 								}

```

Comment:
```suggestion
								double r2 = hollow->InnerFilletRadius.value_or(0.0) * length_factor;
```
This is how you can handle Optional<> lengths, angles, and so on. It condenses the code and makes it more readable.